proposals/lip-0063.md

```diff
@@ -279,6 +279,7 @@ Some new LIPs introduced configurable constants. The values for these constants
 | [LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md#notation-and-constants) | `MAX_ASSET_DATA_SIZE_BYTES` | 18 (The Random Module is the only module adding an entry to the assets property whereby the size of the data property will not exceed 18 bytes) |
 | [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#notation-and-constants), [LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md#notation-and-constants) | `OWN_CHAIN_ID` | `CHAIN_ID_MAINCHAIN` |
 | [LIP 0061](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0061.md#constants) | `BLOCK_TIME` | 10 seconds |
+| [LIP 0061](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0061.md#constants) | `MIN_CERTIFICATE_HEIGHT` | `HEIGHT_SNAPSHOT + 1 + (POS_INIT_ROUNDS + NUMBER_ACTIVE_VALIDATORS - 1) * ROUND_LENGTH +1 = HEIGHT_SNAPSHOT + 1 +  6239741` |
```

Comment:
Why not `HEIGHT_SNAPSHOT +  6239742` in simplified form?

Reply:
I left it as is to be easier for the testing, since this +1 might make the setup more confusing. It is fine for me to switch it, but we should take extra care on testing.

Reply:
Okay, makes sense.